examples/text-generation/run_lm_eval.py

```diff
@@ -34,8 +34,29 @@
 
 
 os.environ.setdefault("TOKENIZERS_PARALLELISM", "false")
+os.environ.setdefault("HF_DATASETS_TRUST_REMOTE_CODE", "true")
```

Comment:
@polisettyvarma to be consistent with the rest of OH examples, why not add trust remote code as an input argument

Reply:
@yafshar done, please take a look now